main/app/attentionRelated/ears/conf/client.ini

```diff
@@ -2,3 +2,4 @@ subsystems          (recog)
 reachingRangeMin    (-0.5 -0.3 -0.15)
 reachingRangeMax    (-0.1 0.3 0.5)
 robot               icub
+zOffset             0.0
```

Comment:
Hi @Tobias-Fischer ,
I am not 100% sure about the quality of z-coordinate of the object perception and also the availability of the connection to `ARE` to get the `table height` when the whole big system runs. For those cases, keeping `zOffset` and its flexibility in modification (through `ini` file) helps to run the demo more safely and  testing modules less exhaustive. 
What is your opinion in this problem @pattacini . Please consider `zOffset` role in the following context of `subSystem_Karma` 

<script src="https://gist.github.com/towardthesea/27cd59676b934cea9997adf1371f7dd4.js"></script>

https://github.com/robotology/wysiwyd/blob/feature/upf/demoBehaviors/main/src/libraries/wrdac/src/subsystems/subSystem_KARMA.cpp#L296-L299